src/specpack.F90

```diff
@@ -16,95 +16,95 @@
 !> @param[out] lcpack length of packed field cpack.
 !>
 !> @author Stephen Gilbert @date 2002-12-19
-subroutine specpack(fld,ndpts,JJ,KK,MM,idrstmpl,cpack,lcpack)
+subroutine specpack(fld, ndpts, JJ, KK, MM, idrstmpl, cpack, lcpack)
   implicit none
 
-  real,intent(in) :: fld(ndpts)
-  integer,intent(in) :: ndpts,JJ,KK,MM
-  integer,intent(inout) :: idrstmpl(*)
-  character(len=1),intent(out) :: cpack(*)
-  integer,intent(out) :: lcpack
+  real, intent(in) :: fld(ndpts)
+  integer, intent(in) :: ndpts, JJ, KK, MM
+  integer, intent(inout) :: idrstmpl(*)
+  character(len=1), intent(out) :: cpack(*)
```

Comment:
`len = 1` to stay consistent in space before and after = sign

Reply:
Fixed.